test/e2e/data/kustomize/common-patches/containerd-kcp.yaml

```diff
@@ -0,0 +1,37 @@
+---
+# This ensures that containerd is installed and configures the system for kubeadm.
+- op: add
+  path: /spec/kubeadmConfigSpec/preKubeadmCommands
+  value:
+  - /usr/local/bin/ci-pre-kubeadm.sh
+- op: add
+  path: /spec/kubeadmConfigSpec/files/-
+  value:
+    content: |
+      #!/bin/bash
+      USE_CI_ARTIFACTS=${USE_CI_ARTIFACTS:=false}
+      if [ ! "${USE_CI_ARTIFACTS}" = true ]; then
+        echo "No CI Artifacts installation, exiting"
+        exit 0
+      fi
+      echo "Installing containerd"
+      apt-get install -y containerd
+    owner: root:root
+    path: /usr/local/bin/ci-pre-kubeadm.sh
+    permissions: "0750"
+- op: add
+  path: /spec/kubeadmConfigSpec/files/-
+  value:
+    content: |
+      br_netfilter
+    path: /etc/modules-load.d/crio-net.conf
+    permissions: "0644"
+- op: add
+  path: /spec/kubeadmConfigSpec/files/-
+  value:
+    content: |
+      net.bridge.bridge-nf-call-iptables = 1
+      net.ipv4.ip_forward = 1
+      net.bridge.bridge-nf-call-ip6tables = 1
+    path: /etc/sysctl.d/99-kubernetes-cri.conf
+    mode: "0644"
```

Comment:
I'm not sure then this is created, but unless it's really early (maybe cloud-init) you may need to restart the 2 systemd services which cause these files to be read.